src/js/controllers/confirm.js

```diff
@@ -340,8 +289,9 @@ angular.module('copayApp.controllers').controller('confirmController', function(
   };
 
   $scope.approve = function(onSendStatusChange) {
-    if ($scope._paypro && $scope.paymentExpired.value) {
-      popupService.showAlert(gettextCatalog.getString('Error'), gettextCatalog.getString('The payment request has expired'));
+
+    if ($scope.paypro && $scope.paymentExpired.value) {
+      popupService.showAlert(null, gettextCatalog.getString('The payment request has expired'));
```

Comment:
the first param for popupService.showAlert is an Optional Title – I think in this context, we're better off not showing a title here. We definitely don't want to say "Error", because it's not an unexpected bug – the payment request has just expired already.

It would be good to polish the sentence though: `The payment request has expired` should be: `This bitcoin payment request has expired.`